frontend/src/app/animals-list/animals-list.component.html

```diff
@@ -1,5 +1,5 @@
-<p>
-  <li *ngFor="let animal of animals">
-    {{animal.id}} - {{animal.title}}
-  </li>
-</p>
+<div class="content">
+  <div class="app-animals-list">
+    <app-animal *ngFor="let animal of animals" [animal]="animal"></app-animal>
```

Comment:
Add `ngFor` to `ng-container` and render `app-animal` component inside of `ng-container`